docs/sdk/src/guides/your_first_pallet/mod.rs

```diff
@@ -21,7 +21,7 @@
 //! this guide, namely the crate/package names, based on which template you use.
 //!
 //! > Be aware that you can read the entire source code backing this tutorial by clicking on the
-//! > [`source`](./mod.rs.html) button at the top right of the page.
+//! > `source` button at the top right of the page.
```

Comment:
This link didn't work.
It is however possible to make it work - it would look like this:

```
[`source`](../../../src/polkadot_sdk_docs/guides/your_first_pallet/mod.rs.html)
```

(Unless I'm missing some other obvious way of linking to source)
I thought it's better to not have a link, since the user is pointed to the built-in button anyway.

Reply:
I am surprised, why didn't the original one, or sth like `../mod.rs` work? I don't see why we have to back up 3 steps and re-navigate ourselves into the same directory?

Reply:
> back up 3 steps and re-navigate ourselves into the same directory

Not the same directory.

It navigates from:
`target/doc/polkadot_sdk_docs/guides/your_first_pallet/index.html`
to:
`target/doc/src/polkadot_sdk_docs/guides/your_first_pallet/mod.rs.html`

(note added `src`)

So it will link from [here](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/guides/your_first_pallet/index.html) to [here](https://paritytech.github.io/polkadot-sdk/master/src/polkadot_sdk_docs/guides/your_first_pallet/mod.rs.html).

Reply:
Okay, we can remove the link for now. 

As a wishlist item, I wish we could amend each page to have a few fixed buttons/links in a more pronounced way: 

- how to contribute to this page
- how to see the source of this page

The latter is already visible via the default rust HTML indeed.